startupscript/dataproc/startup.sh

```diff
@@ -935,6 +947,58 @@ fi
     ${RUN_PYTHON} ${HAIL_SCRIPT_PATH}
   fi
 
+  #################################
+  # Configure Proxy Agent Overrides
+  #################################
+
+  # Map the CLI server to the appropriate UI url
+  case $TERRA_SERVER in
+    verily-devel)
+      ui_base_url=terra-devel-ui-terra.api.verily.com
+      ;;
+    verily-autopush)
+      ui_base_url=terra-autopush-ui-terra.api.verily.com
+      ;;
+    verily-staging)
+      ui_base_url=terra-staging-ui-terra.api.verily.com
+      ;;
+    verily-preprod)
+      ui_base_url=terra-preprod-ui-terra.api.verily.com
+    ;;
+    verily)
+      ui_base_url=workbench.verily.com
+      ;;
+    *)
+      >&2 echo "ERROR: $TERRA_SERVER is not a known verily server."
+      exit 1
+      ;;
+  esac
+
+  # Insert a link element to render
+  readonly WORKSPACE_LINK_EL="<a id=\"workspace\" class=\"forum\" target=\"_blank\" href=\"https://${ui_base_url}/workspaces/${TERRA_WORKSPACE}\">${TERRA_WORKSPACE}</a>"
+  sed -i "s/<banner-title>/<banner-title>\n$WORKSPACE_LINK_EL \&gt; /" ${PROXY_AGENT_BANNER}
```

Comment:
I think it's preferable to use single quotes where possible, since we are often trying to emit double-quotes.
Also suggest using `#` instead of `/` since we're more likely to want to emit path symbols than hashes.

```
sed -i 's#<banner-title>#<banner-title>\n'"${WORKSPACE_LINK_EL}"' \&gt; #' "${PROXY_AGENT_BANNER}"
```